src/main/java/chess/domain/piece/implement/BlackPawn.java

```diff
@@ -0,0 +1,15 @@
+package chess.domain.piece.implement;
+
+import chess.domain.board.Path;
+import chess.domain.piece.Color;
+
+public class BlackPawn extends Pawn {
+    public BlackPawn() {
+        super(Color.BLACK);
+    }
+
+    @Override
+    boolean isBackward(Path path) {
```

Comment:
```suggestion
    protected boolean isBackward(Path path) {
```
아니면 접근제어자가 default인 이유가 있나요

Reply:
인텔리제이에서 자동완성된 메서드를 사용하면서 고민하지 않고 놓쳤던 부분입니다...!
부모 클래스에서는 접근을 하되, 다른 클래스에서는 사용하지 못하도록 `protected` 붙여주겠습니다!